src/components/nested-navigation/nested-navigation.config.yml

```diff
@@ -0,0 +1,3 @@
+title: Nested Navigation
```

Comment:
After thinking about it I'm not sure if _Nested navigation_ is necessarily the most appropriate name. In theory, you could use this component as a side navigation that only had one level of non-nested links.

This is fine for now, but let's talk about his more on Monday.

Reply:
We talked through this offline. We'll name this component  to _Sidebar_, since that will be its main intent.